packages/web-config-server/src/apiV1/measureBuilders/helpers.js

```diff
@@ -74,3 +74,17 @@ export const analyticsToMeasureData = (analytics, customDataKey) =>
     [customDataKey || dataElement]: value,
     submissionDate: periodToMoment(period.toString()).format('YYYY-MM-DD'),
   }));
+
+export const groupAnalyticsByOrgUnit = analytics => {
+  const analyticsByOrgUnit = {};
+
+  analytics.forEach(analytic => {
+    if (!analyticsByOrgUnit[analytic.organisationUnit]) {
+      analyticsByOrgUnit[analytic.organisationUnit] = [];
+    }
+
+    analyticsByOrgUnit[analytic.organisationUnit].push(analytic);
+  });
+
+  return analyticsByOrgUnit;
+}
```

Comment:
I think this is equivalent to:

```js
import 'groupBy' from 'lodash.groupby';

const analyticsByOrgUnit = groupBy(analytics, 'organisationUnit');
```

in which case there is no point to define it as a new method

Reply:
my bad, that was a bit silly

Reply:
No worries man!